mailbox/api/src/test/java/org/apache/james/mailbox/manager/ManagerTestResources.java

```diff
@@ -71,7 +71,7 @@ public ManagerTestResources(IntegrationResources<T> integrationResources) throws
         quotaRootResolver = integrationResources.createQuotaRootResolver(mailboxManager);
         quotaManager = integrationResources.createQuotaManager(maxQuotaManager, mailboxManager);
         integrationResources.init();
-        session = mailboxManager.login(USER, USER_PASS);
+        session = mailboxManager.createSystemSession(USER);
```

Comment:
I don't understand why it's better: if you login, you are reproducing a real behaviour, while creating a system session you are not. No?

Reply:
Well, no need to customize authentication logic and add a fake user when it is only used in test setUp?

Reply:
if IRL people get sessions from login method, we should use login method in test.